hw2/03-count.js

```diff
@@ -1,3 +1,21 @@
-// Add your code here
+// // Add your code here
+const inputWord = document.querySelector("#inputWord");
+const textContainer = document.querySelector("#textContainer");
 
-input.addEventListener('keydown', handleKeyDown);
+inputWord.addEventListener("keyup", () => {
+  const searchText = inputWord.value.trim();
+  const textContent = textContainer.textContent;
+  if (searchText != "") {
```

Comment:
According to the Airbnb Style, we should use "!==" instead of "!=" when we want to compare things to see if they are strictly not equal.

Reply:
Thanks for pointing it out.